code/game/gamemodes/game_mode.dm

```diff
@@ -271,7 +271,7 @@
 			if(player.client && player.ready)
 				if(!player.mind in drafted || !player.mind in candidates) // Players were getting placed in candidates AND drafted lists.
 					if(player.client.desires_role(role, display_to_user=poll)) // We don't have enough people who want to be antagonist, make a seperate list of people who don't want to be one
-						if(!jobban_isbanned(player, "Syndicate") && !jobban_isbanned(player, role)) //Nodrak/Carn: Antag Job-bans
```

Comment:
As far as I can tell, the Syndicate ban is what happens if you go in and click on this button in the jobban panel:
![image](https://cloud.githubusercontent.com/assets/5942183/21433756/62ee3926-c871-11e6-951d-d04ed41d95d5.png)
I don't recall exactly if that's how I banned the guy I did for going AFK roundstart as cult, but I can't think of any other way it could've happened. Does it not just list him as banned from Syndicate and not banned from all the roles in there?